### PR TITLE
populate map with nearby pokemon

### PR DESCRIPTION
**Pull request by rohanjain1**
-can fetch and display location of nearby pokemon, using current location
-pogo API implemented
-images for pokemon added
-Using async tasks for quick and dirty tests[ BAD rohan! ]

### CODE REVIEW

File: app/src/main/java/com/omkarmoghe/pokemap/MainActivity.java [hunk 1]
```diff
@@ -94,6 +121,31 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                    public boolean verify(String s, SSLSession sslSession) {
+                        return true;
+                    }
+                })
+                .cookieJar(new PersistentCookieJar(new SetCookieCache(), new SharedPrefsCookiePersistor(getApplicationContext())))
+                .followRedirects(false)
+                .followSslRedirects(false)
+                .build();
+
+                //new GoogleLogin(httpClient).login("token");
```
Comment (CanOrhan) [inline app/src/main/java/com/omkarmoghe/pokemap/MainActivity.java:136]: Don't need commented code version controlled

File: app/src/main/java/com/omkarmoghe/pokemap/MainActivity.java [hunk 2]
```diff
@@ -222,7 +277,52 @@ public void onResponse(Call call, Response response) throws IOException {
...
      * See https://github.com/AHAAAAAAA/PokemonGo-Map/blob/master/example.py#L378
      */
-    private void getPokemon() {
+
+    private void getPokemon(){
+        Log.d("getPokemon","trying to get catchable Pokemon");
+        AsyncTask<Object,Object,Object>asyncTask = new AsyncTask() {
+            @Override
+            protected Object doInBackground(Object[] objects) {
+                /*try {
```
Comment (CanOrhan) [inline app/src/main/java/com/omkarmoghe/pokemap/MainActivity.java:286]: Commented code.

**Comment by Robbievan10** [comment]
@rohanjain1 how can i clone the files you changed? i would like to test the changes you made.

**Comment by filfat** [comment]
You'll need to add the license to the travis.yml file for the build to succeed. See https://github.com/omkarmoghe/Pokemap/blob/dev/.travis.yml#L13

**Pull request merged**